### ISSUE TITLE
No Sensor data available

### ISSUE THREAD

**Issue by Joakim-Wallman** [home-assistant/home-assistant.github.io#24197]
### Feedback

I'm not really sure how it is supposed to work, but it looks like something is wrong.
The Tibber integration shows my full name, which comes from Tibber and I can send notifications to my Tibber App, so the tibber tag seem to work as it should.
But I can't find any related Sensors anywhere.
Is the tibber integration supposed to show any services or entyties? It does not show any for me.
I tried to to use the developer tools tab and enter "sensor." and get several suggestions, but nothing is related to tibber or power usage.
The tibber integration diagnostic file have a data field containing information about my home, but those fields are not really connected to me. It is e.g. saying that I do not have an active subscription. I tried to fetch home information using the tibber api directly which contains my full address etc, is the diagnostic file supposed to show the same information?
I can also get electric prices and consumption using the tibber api directly.

If you help me to get this working I can help out to improve the tibber integration description page with my own experience.

### URL

https://www.home-assistant.io/integrations/tibber/

### Version

2022.9.5

**Comment by Joakim-Wallman** [comment]
I still have this problem.
I have got confirmation from other users that is not supposed to be like this.
Could it be because I'm not using HAOS? (I can't really see why, but it is a difference from a user that got it working).
@Danielhiversen have I filed this problem in the wrong location? Maybe it belongs to core.

**Comment by Joakim-Wallman** [comment]
I think I found a major reason for my problem. tibber.py was not installed.
For others reading this, you can find more info at https://pypi.org/project/tibber.py/
But it still fails, __init__py row 42 contains a row calling tibber.Tibber(), but if I understand it correct there is no entry called Tibber()

**Comment by Joakim-Wallman** [comment]
I thought I was onto something in my last comment.
I tried to run tibber from python command prompt and could see that tibber.Tibber did not exist.
But after installing pyTibber on my private account I could see tibber.Tibber.
Changing to HomeAssistants own python environment showed pyTibber version 0.24, but no tibber.Tibber().
Updated pyTibber to version 0.26, success, now tibber.Tibber is there.

But still no Entities for tibber in HA GUI. No error messages anymore, so how do I continue?
